src/main/java/com/cegeka/academy/service/userChallenge/UserChallengeService.java

```diff
@@ -28,4 +28,6 @@ UserChallenge rateUser(UserChallengeDTO userChallengeDTO, Long ownerId)
     UserChallenge initUserChallenge (Challenge challenge, Invitation invitation) throws NotFoundException;
 
     List<ChallengeDTO> getNextChallengesForAnUser(Long userId) throws NotFoundException;
+
+    List<ChallengeDTO> getChallengesWithPendingStatusForInvitation(Long userId) throws NotFoundException;
```

Comment:
Hai sa modificam aceasta metoda, sa mergem pe getChallengeByInvitationStatus(Long userId, Status status). In felul acesta metoda se poate utiliza pentru orice status.